controller/cdr/queueSummaryHourlyCtrl.js

```diff
@@ -342,7 +367,14 @@
 
                 if($scope.skillFilter && $scope.skillFilter.length > 0)
                 {
-                    buildSummaryListByHr($scope.obj.day, 1, $scope.skillFilter[0].QueueName, $scope.skillFilter[0].RecordID, momentTz, function (err, processDoneResp)
+                    skillArr = [];
```

Comment:
Always declare variable with let or var to stop getting variables hoisted to global scope unintentionally